teos/src/config.rs

```diff
@@ -242,6 +242,27 @@ impl Config {
     pub fn is_default(&self) -> bool {
         self == &Config::default()
     }
+
+    /// Logs non-default options.
+    pub fn log_non_default_options(&self) {
+        let json_default_config = serde_json::json!(&Config::default());
+        let json_config = serde_json::json!(&self);
+        let protected_keys = ["btc_rpc_user", "btc_rpc_password"];
```

Comment:
nit: Let's call this `sensitive_args` now that we know that's how core calls them.

Reply:
> nit: Let's call this `sensitive_args` now that we know that's how core calls them.

**protected_keys** has to be changed to **sensitive_args**?

Reply:
Yes

Reply:
> Yes

done